README.rst

```diff
@@ -67,7 +67,7 @@ you're good to go:
 
 .. code-block:: bash
 
-    $ export PATH="$PATH:$HOME/.composer/vendor/bin"
+    $ export PATH="$PATH:$HOME/.config/composer/vendor/bin"
```

Comment:
According to [Composer doc](https://getcomposer.org/doc/03-cli.md#composer-home), the default config directory varies depending on the Unix system you are using and both values are valid actually. IMO it would be better to say this is just an example and add this doc link to the README.

Anyway, if we update the `code-block`, we should also update the sentence above as it mentions the path too.